### PR TITLE
44 fr roadchoice

### LINKED ISSUE #44: FR: [RoadChoice] Cards variant for RoadChoice.com
**Description**

On the production [Resources page](https://roadchoice.com/resources), there is a cards list to show the different resources available on the website.
The goal of this ticket is to implement a card variant for road choice, in the brand styling, to be used in this context.
The card should allow the addition of an svg of a png with transparent bg.

Image

**Acceptance criteria**
- The card variant for RoadChoice is styled as per brand's design (colour scheme / font)
- The block allows editors to add an svg or a png with transparent bg and a text
- On smaller viewports, 2-cols should change to 1-col

### PR DESCRIPTION
**Pull request by Sujata928573**
Fix  #44 

Test URLs:
- Before: https://main--vg-roadchoice-com--hlxsites.hlx.page/drafts/shomps/cards
- After: https://44-fr-roadchoice--vg-roadchoice-com--hlxsites.hlx.page/drafts/shomps/cards

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -1,31 +1,86 @@
...
+	 position: absolute;
+	 bottom: 0;
+	 left: 0;
+	 width: 100%;
+	 height: 100%;
+	 display: flex;
+	 align-items: flex-end;
+	 justify-content: center;
+   font-size: 16px;
+}
```
Comment (markovukiceviccn) [inline blocks/cards/cards.css:18]: Code above needs proper formatting and indentation.
Check your IDE settings for proper code formatting and indentation.

Comment (markovukiceviccn) [reply blocks/cards/cards.css:18]: In `.cards li a.primary` selector, the `a` is not necessary. 
Hint: tag names are less performant compared to class names ;)

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -1,31 +1,86 @@
...
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
 }
 
 .cards .cards-card-body {
-  margin: 16px;
+  font-weight: bold;
+  font-size: 16px;
```
Comment (markovukiceviccn) [inline blocks/cards/cards.css:46]: Shouldn't we avoid `font-weight` property?

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -1,31 +1,86 @@
...
-  width: 100%;
-  aspect-ratio: 4 / 3;
-  object-fit: cover;
+li.li_list:nth-child(odd) {
+  float: right;
+}
+
+li.li_list:nth-child(even) {
+  float: left;
 }
```
Comment (markovukiceviccn) [inline blocks/cards/cards.css:67]: A perfect scenario for nesting ;)

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -1,31 +1,86 @@
...
+	 left: 0;
+	 width: 100%;
+	 height: 100%;
+	 display: flex;
+	 align-items: flex-end;
+	 justify-content: center;
+   font-size: 16px;
+}
+.cards > ul > li img {
+  aspect-ratio: 0;
```
Comment (markovukiceviccn) [inline blocks/cards/cards.css:20]: probably an unnecessary over-specificity.

Comment (jonatan-lledo-netcentric) [reply blocks/cards/cards.css:20]: or better, add by js a class to reduce the selector scope

File: blocks/cards/cards.js [hunk 5]
```diff
@@ -1,18 +1,15 @@
-import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
-
```
Comment (jonatan-lledo-netcentric) [inline blocks/cards/cards.js:2]: please, add createElement from scripts.js:
`import { createElement} from '../../scipts/scripts.js'`
and refactor all the `document.createElement` with our custom function.
thanks

**Review by jonatan-lledo-netcentric — CHANGES_REQUESTED** [review]
the whole card has to be a link to become a sort of a button, so it needs to use DOM manipulation to set all the structure to be clickable

File: blocks/cards/cards.css (hunk 3 above)
Comment (jonatan-lledo-netcentric) [reply blocks/cards/cards.css:71]: at least not now for this project but we will consider to start using CSS nesting

**Review by jonatan-lledo-netcentric — APPROVED** [review]

**Pull request merged**